### PR TITLE
Conditional wheel events subs

### PR DESCRIPTION
**Pull request by SlicedSilver**
**Type of PR:** enhancement

**PR checklist:**

- [x] Includes tests
- [ ] Documentation update

**Problem:**
The library unconditionally subscribes to wheel events but with `_options.handleScroll.*` turned off this handler is just two math operations and an if statement. But even an empty no-op wheel handler can slow down the page scrolling.

**Overview of change:**

- Conditionally add or remove the 'wheel' event listener based on the `handleScroll/mouseWheel` and `handleScale/mouseWheel` chart options.
- Added new e2e test suite for interactions.

**Is there anything you'd like reviewers to focus on?**
- Check that CI pipeline still works correctly with new check for interactions tests.

### CODE REVIEW

File: tests/e2e/interactions/test-cases/mouse-wheel/default-scroll-handler.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		});
+
+		time.setUTCDate(time.getUTCDate() + 1);
+	}
+	return res;
+}
+
+function interactionsToPerform() {
+	return ['scrollLeft', 'scrollDown'];
+}
```
Comment (romfrancois) [inline tests/e2e/interactions/test-cases/mouse-wheel/default-scroll-handler.js:17]: Can't we put at least those 2 functions in a utils.js file vs duplicating the code?

File: tests/README.md [hunk 2]
```diff
@@ -107,3 +107,21 @@ Alternatively, you can run the test on a specific file like this:
 ```bash
 node ./tests/e2e/memleaks/runner.js ./dist/lightweight-charts.standalone.development.js
 ```
+
+### Interactions
+
+The interactions tests check whether the library is correctly handling user interaction on the chart. Interactions include: mouse scrolling, mouse dragging, and touches.
```
Comment (romfrancois) [inline tests/README.md:113]: typo with `interactionS`?  - "...handling user interaction on the chart."

**Comment by SlicedSilver** [comment]
Since #1166 has now been merged, this PR is ready to be reviewed and merged.

**Review by romfrancois — DISMISSED** [review]
lgtm - thanks for bringing in e2e tests for this one!

**Review by kirchet — APPROVED** [review]

**Pull request merged**